### DISCUSSION TITLE
1.5.0.2

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#469] [Обновления]
## Список изменений
- Улучшена очистка описания на Youtube от @SashaXser в #467

## What's Changed
* Улучшение очиски описания на Youtube by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/467
* improve some russian translation texts by @Blueberryy in https://github.com/ilyhalight/voice-over-translation/pull/465
* Merged dev by @ilyhalight in https://github.com/ilyhalight/voice-over-translation/pull/468

## New Contributors
* @Blueberryy made their first contribution in https://github.com/ilyhalight/voice-over-translation/pull/465

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.5.0.1...1.5.0.2

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.0.2'>1.5.0.2</a>.</em>

**Comment by SrevSA** [comment]
Последняя версия ,не знаю, возможно это только у меня, но перевод отстает от картинки. Точнее проматывая видео, раньше я получал синхронный перевод ,то сейчас голос перевода идет  с начала ,на каком бы этапе не было видео.

**Reply by ilyhalight** [reply]
у меня все работает

**Reply by SrevSA** [reply]
Возможно это связано с блокировкой Яндекс. Я попробую из под VPN  Жаль что не найдено решение по понятным и независящим от вас причинам перевода на язык отличный от русского.
